ros/src/inertial_sense_ros.cpp

```diff
@@ -1168,8 +1201,13 @@ void InertialSenseROS::INS4_callback(eDataIDs DID, const ins_4_t *const msg)
                 msg_odom_ned.twist.twist.angular.x = result[0];
                 msg_odom_ned.twist.twist.angular.y = result[1];
                 msg_odom_ned.twist.twist.angular.z = result[2];
-                rs_.odom_ins_ned.pub.publish(msg_odom_ned);
+                if (rs_.odom_ins_ned.pub)
+                {
+                    rs_.odom_ins_ned.pub.publish(msg_odom_ned);
 
+                }
+                
+                
```

Comment:
Let's keep for formatting consistent.  if its one empty line after a closing brace everywhere else in the source file, we should keep it to one empty line after a closing brace here too.